.spelling

```diff
@@ -741,6 +741,7 @@ Youtube
 Zack
 Zhang
 Zipkin
+CronJob
```

Comment:
Additions to .spelling would be in alphabetical order. I believe cron job is normally two words.